### ISSUE TITLE
Extend Generic Executor to send events back to keptn after scripts executed

### ISSUE THREAD

**Issue by grabnerandi** [keptn-sandbox/generic-executor-service#3]
Use Case:
If we want to use e.g: a Python script for delivery we would create a script that gets executed for a configuration.change event.
Once the script is done with the delivery it needs to send the "deployment.finished" event back to Keptn.
Right now this is not easily possible as the script doesnt have the Keptn's API Endpoint. We should provide the endpoint and maybe some helper functions for Python and bash so that scritp developers can easily send an event to Keptn at the end of the action execution

**Comment by christian-kreuzberger-dtx** [comment]
No longer needed as we are deprecating this service https://github.com/keptn-sandbox/generic-executor-service/issues/16
